src/stratis_cli/_actions/_top.py

```diff
@@ -32,6 +33,38 @@
 from ._formatting import TABLE_FAILURE_STRING, fetch_property, print_table
 
 
+def _generate_pools_to_blockdevs(managed_objects, to_be_added, tier):
+    """
+    Generate a map of pools to which block devices they own
+    :param managed_objects: the result of a GetManagedObjects call
+    :type managed_objects: dict of str * dict
```

Comment:
There's a couple of missing param documentation lines.

Reply:
Resolved.